cms/static/js/views/course_outline.js

```diff
@@ -141,16 +141,21 @@ define(["jquery", "underscore", "js/views/xblock_outline", "common/js/components
 
             editXBlock: function() {
                 var enable_proctored_exams = false;
+                var enable_timed_exams = false;
                 if (this.model.get('category') === 'sequential' &&
                     this.parentView.parentView.model.has('enable_proctored_exams')) {
-
                     enable_proctored_exams = this.parentView.parentView.model.get('enable_proctored_exams');
                 }
+                if (this.model.get('category') === 'sequential' &&
```

Comment:
This check ` if (this.model.get('category') === 'sequential'` is duplicated. Move `if ( this.parentView.parentView.model.has('enable_timed_exams')) {` block inside the first one